crates/codegen/runtime/cargo/crate/src/runtime/compilation/internal_builder.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(feature = "__private_wasm_apis")]
```

Comment:
I think this should be used by both languages' `CompilationUnitBuilder` in both languages?